client/FxAccountClient.js

```diff
@@ -1031,4 +1099,30 @@ define([
   return FxAccountClient;
 });
 
+var METRICS_CONTEXT_OPTIONS = {
+  context: true,
+  entrypoint: true,
+  migration: true,
+  service: true,
+  utmCampaign: true,
```

Comment:
Over in the content server, [these fields come from the query parameters](https://github.com/mozilla/fxa-content-server/pull/3587/files#diff-3a32567cd739e05b2dc5a399f75ab73fR15) and are in snake case. I assume the auth server will convert case for heka output?